### PR TITLE
Scp 865

### PR DESCRIPTION
**Pull request by babyboucher**
Lets you kill with no trace and shot sound

### CODE REVIEW

File: CustomItems/Items/Scp865.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+        /// <inheritdoc/>
+        public override uint ClipSize { get; set; } = 16;
+
+        /// <inheritdoc/>
+        protected override void OnShooting(ShootingEventArgs ev)
+        {
+            ev.Shooter.SetWeaponAmmo(ev.Shooter.CurrentItem, (int)ev.Shooter.CurrentItem.durability - 1);
+            Player target = Player.Get(ev.Target);
+            if (target != null)
```
Comment (galaxy119) [inline CustomItems/Items/Scp865.cs:64]: There's no FF check. The shooting event will still fire with the target being a friendly player, even if server FF is off.

**Review by galaxy119 — CHANGES_REQUESTED** [review]

File: CustomItems/Items/Scp865.cs (hunk 1 above)
Comment (babyboucher) [reply CustomItems/Items/Scp865.cs:64]: Yes, that's the point

Comment (babyboucher) [reply CustomItems/Items/Scp865.cs:64]: Actually, when I get home I will make this more balanced, if you posses the gun you can use it to tk and also you can be tked if you shoot a teammate

Comment (galaxy119) [reply CustomItems/Items/Scp865.cs:64]: Ah, ok, I didn't realize that was by design.

**Review by galaxy119 — CHANGES_REQUESTED** [review]
Please update NuGet reference to EXILED 2.9.0-beta and set override property "ShouldMessageOnGban" to true.

Reason: Could be seen as anti-recoil/anti-sound hack.

**Pull request closed without merging**